packages/cli/src/prompts/prompt.ts

```diff
@@ -220,6 +221,8 @@ export function methodsList(
       return { name: label, value: { name, selector } };
     },
   );
+
+  return sortBy(methods, ['name']);
```

Comment:
A name starting with a dollar sign [is a valid identifier in Solidity](https://github.com/solidityj/solidity-antlr4/blob/edce5c44d2d681e32bb23034d822bf0fcf1cb43e/Solidity.g4#L419-L421), and `$ < *` lexicographically, so any functions starting with dollar would be listed _before_ the initializable ones.

```js
> ['*', '$', 'A', 'a'].sort()
[ '$', '*', 'A', 'a' ]
```

Reply:
ups, good catch!